src/stage1/forward.jl

```diff
@@ -1,14 +1,14 @@
-partial(x::TangentBundle, i) = x.partials[i]
-partial(x::TaylorBundle{1}, i) = x.coeffs[i]
-partial(x::UniformBundle, i) = x.partial
+partial(x::ExplicitTangentBundle, i) = x.tangent.partials[i]
+partial(x::TaylorBundle{1}, i) = x.tangent.coeffs[i]
+partial(x::UniformBundle, i) = x.tangent.val
 partial(x::CompositeBundle{N, B}, i) where {N, B} = Tangent{B}(map(x->partial(x, i), x.tup)...)
 partial(x::ZeroTangent, i) = ZeroTangent()
 primal(x::AbstractTangentBundle) = x.primal
 primal(z::ZeroTangent) = ZeroTangent()
 
-first_partial(x::TangentBundle{1}) = getfield(getfield(x, :partials), 1)
-first_partial(x::TaylorBundle{1}) = getfield(getfield(x, :coeffs), 1)
-first_partial(x::UniformBundle) = getfield(x, :partial)
+first_partial(x::ExplicitTangentBundle{1}) = getfield(getfield(getfield(x, :tangent), :partials), 1)
+first_partial(x::TaylorBundle{1}) = getfield(getfield(getfield(x, :tangent), :coeffs), 1)
+first_partial(x::UniformBundle) = getfield(getfield(x, :tangent), :val)
 first_partial(x::CompositeBundle) = map(first_partial, getfield(x, :tup))
```

Comment:
Do we actually need to define `first_partial` separately? Can't we just use `partial(x, 1)` instead?